ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -2,8 +2,7 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
-using KSP.Localization;
-
+using UnityEngine;
```

Comment:
I think we have been trying to avoid that in Principia, fully-qualifying the UnityEngine stuff instead.